go-apps/meep-ctrl-engine/server/ctrl-engine.go

```diff
@@ -1245,3 +618,155 @@ func ceGetStates(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 	fmt.Fprint(w, string(jsonResponse))
 }
+
+// ------------------
+
+func sendEventNetworkCharacteristics(event ceModel.Event) (string, int) {
+	if event.EventNetworkCharacteristicsUpdate == nil {
+		return "Malformed request: missing EventNetworkCharacteristicsUpdate", http.StatusBadRequest
+	}
+
+	// elementFound := false
+	netChar := event.EventNetworkCharacteristicsUpdate
+
+	err := activeModel.UpdateNetChar(netChar)
+	if err != nil {
+		return err.Error(), http.StatusInternalServerError
+	}
+	return "", -1
+}
+
+func sendEventMobility(event ceModel.Event) (string, int) {
+	if event.EventMobility == nil {
+		return "Malformed request: missing EventMobility", http.StatusBadRequest
+	}
+	// Retrieve target name (src) and destination parent name
+	elemName := event.EventMobility.ElementName
+	destName := event.EventMobility.Dest
+
+	oldNL, newNL, err := activeModel.MoveNode(elemName, destName)
+	if err != nil {
+		return err.Error(), http.StatusInternalServerError
+	}
+	log.WithFields(log.Fields{
+		"meep.log.component": "ctrl-engine",
+		"meep.log.msgType":   "mobilityEvent",
+		"meep.log.oldLoc":    oldNL,
+		"meep.log.newLoc":    newNL,
```

Comment:
integration issue.. frontend refers to oldPoa and newPoa for display.. need to update here back or meep-metrics-engine.

Reply:
This change is included and required only in the frontend-dev branch that will be merged shortly.